pixi.toml

```
@@ -173,6 +171,8 @@ tomli-w = "*"
 xmipy = "*"
 xugrid = "*"
 xarray = "*"
+ipykernel = ">=6.29.3,<6.30"
+networkx = ">=3.3,<4"
```

```suggestion
networkx = ">=3.3"
```

I'd say let's leave the upper bound off for now. It's not there for other dependencies, and we have no automated tooling to notify us of new breaking releases like dependabot set up for pixi.toml. And when updating the pixi.lock we rely on our tests.